ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts

```diff
@@ -62,7 +62,7 @@ export class AnalysisContextFormComponent extends FormComponent
         },
         {
             "id": 100,
-            "name": "Migration to Red Hat JBoss EAP 6",
+            "name": "Migration to JBoss EAP 6",
```

Comment:
again this should be in sync with https://github.com/windup/windup-web/blob/master/services/src/main/resources/migration-paths/migration-paths.xml